### PR TITLE
AAP-35099: adds info for token management with PAH and keycloak

### PR DESCRIPTION
**Pull request by hherbly**
See [AAP-35099](https://issues.redhat.com/browse/AAP-35099) for more info

### CODE REVIEW

**Review by rochacbruno — APPROVED** [review]

File: downstream/assemblies/hub/assembly-hub-create-api-token.adoc [hunk 1]
```diff
@@ -10,10 +10,12 @@ Before you can interact with {HubName} by uploading or downloading collections,
 
 Your method for creating the API token differs according to the type of {HubName} that you are using:
 
-* {HubNameStart} uses Offline token management. See xref:proc-create-api-token[Creating the API token in {HubName}].
+* {HubNameStart} uses Offline token management. See xref:proc-create-api-token[Creating the Offline token in {HubName}].
```
Comment (dcdacosta) [inline downstream/assemblies/hub/assembly-hub-create-api-token.adoc:13]: I believe Offline should be lowercase to offline.

File: downstream/assemblies/hub/assembly-hub-create-api-token.adoc [hunk 2]
```diff
@@ -10,10 +10,12 @@ Before you can interact with {HubName} by uploading or downloading collections,
...
 
 * {PrivateHubNameStart} uses API token management. See xref:proc-create-api-token-pah[Creating the API token in {PrivateHubName}].
 
+* If you are using Keycloak to authenticate your {PrivateHubName}, follow the procedure for xref:proc-create-api-token[Creating the Offline token in {HubName}].
```
Comment (dcdacosta) [inline downstream/assemblies/hub/assembly-hub-create-api-token.adoc:17]: Suggest changing Offline to offline.

**Review by dcdacosta — APPROVED** [review]
Two minor suggestions, otherwise, LGTM!

**Review by dcdacosta — APPROVED** [review]
LGTM!

**Pull request merged**